iFixit/react-commerce
# Piwik: Add Category Data To Some Ecommerce Events

## Overview

We want to start adding category data to piwik ecommerce events as specified in the connected issue. This takes the category data added to the ifixit apis by https://github.com/iFixit/ifixit/pull/51401 and uses it throughout react commerce. It is pretty much just passing the data through to piwik ecommerce events.

NOTE: The only ecommerce events that this doesn't cover/add the category data to is anything to do with the `Frequently Bought With` products. That will have to be covered in a later pull.

## QA

Do all events except those related to the `Frequently Bought Together` now send the correct category data to piwik?
Make sure that youre calling to an api that has https://github.com/iFixit/ifixit/pull/51526 merged

Connects: https://github.com/iFixit/ifixit/issues/51220
